### PR TITLE
Add Recharts

### PR DESCRIPTION
**Pull request by ysm-dev**
https://recharts.org/

- https://recharts.org/
- https://recharts.org/en-US/examples

### CODE REVIEW

File: src/technologies/r.json [hunk 1]
```diff
@@ -705,6 +705,18 @@
...
     "website": "https://rechargepayments.com"
   },
+  "Recharts": {
+    "cats": [
+      25
+    ],
+    "description": "Recharts is a Redefined chart library built with React and D3.",
+    "icon": "Recharts.png",
+    "dom": [
+      "[class^=recharts-]"
```
Comment (nurbek91) [inline src/technologies/r.json:715]: Maybe replace with this 
examples:
https://coinloop.io/
https://coinmetro.com/
https://www.uvocorp.com/
```
"dom": {
      "div > div > svg.recharts-surface": {
        "attributes": {
          "class": ""
        }
      }
    },
```

Hey @AliasIO ,  Does this design help with broad matches and does it improve performance?

File: src/technologies/r.json [hunk 2]
```diff
@@ -705,6 +705,18 @@
     ],
     "website": "https://rechargepayments.com"
   },
+  "Recharts": {
+    "cats": [
+      25
+    ],
+    "description": "Recharts is a Redefined chart library built with React and D3.",
```
Comment (nurbek91) [inline src/technologies/r.json:712]: ` "description": "Recharts is a component-based charting library, which is exclusively built for React applications."`

`"implies": "React",` ?

File: src/technologies/r.json [hunk 3]
```diff
@@ -705,6 +705,18 @@
...
+    "icon": "Recharts.png",
```
Comment (nurbek91) [inline src/technologies/r.json:713]: I drew two versions of the icon. One is more readable but doesn't match the actual icon. In general, there is no such icon there, just text. Large text is not very readable. Generally choose what suits  best.
Recharts1
Recharts

**Review by nurbek91 — CHANGES_REQUESTED** [review]
Hey mate, thanks for PR!!!

File: src/technologies/r.json (hunk 1 above)
Comment (AliasIO) [reply src/technologies/r.json:715]: Yes it does, although the original is ok. As long as the number of elements that match the pattern is limited.

File: src/technologies/r.json (hunk 2 above)
Comment (ysm-dev) [reply src/technologies/r.json]: [3c1e744](https://github.com/wappalyzer/wappalyzer/pull/6956/commits/3c1e744ee53f74da981b400ded14d06d8195c084)
[f54528c](https://github.com/wappalyzer/wappalyzer/pull/6956/commits/f54528cbbf2195c0223c46bda8da0820b42a56bf)

File: src/technologies/r.json (hunk 3 above)
Comment (ysm-dev) [reply src/technologies/r.json]: [aff59ee](https://github.com/wappalyzer/wappalyzer/pull/6956/commits/aff59ee85cc75fb63eed5b97d669fc52bc4ba834)

**Review by nurbek91 — APPROVED** [review]
Thanks mate!!

**Pull request merged**